lib/uploadcare/rails/api/rest/base.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'uploadcare'
+
+module Uploadcare
+  module Rails
```

Comment:
Are this API really Rails-specific?

Reply:
No, this is just a wrapper. Reason, why this API is here - convenience for Rails-users.